context_test.go

```diff
@@ -341,49 +334,49 @@ func TestDetectProjectGOPATH(t *testing.T) {
 		{
 			name:         "project-with-no-AbsRoot",
 			root:         "",
-			resolvedRoot: filepath.Join(ctx.GOPATHs[0], "src", "real", "path"),
+			resolvedRoot: filepath.Join(ctx.GOPATHs[0], "src/real/path"),
```

Comment:
Just trying to clarify, shouldn't [this line](https://github.com/golang/go/blob/master/src/path/filepath/path.go#L159) do that?